### PR TITLE
[NEW] Internal Hubot load scripts from custom folders

### PR DESCRIPTION
**Pull request by ggazzo**
@RocketChat/core

### CODE REVIEW

**Comment by sampaiodiego** [comment]
I think we need to add a `[BREAK]` because it will break to anyone who are using official hubot scripts.

@RocketChat/core what you think?

File: packages/rocketchat-internal-hubot/hubot.js [hunk 1]
```diff
@@ -0,0 +1,242 @@
+/* eslint-disable no-use-before-define no-unused-vars */
```
Comment (sampaiodiego) [inline packages/rocketchat-internal-hubot/hubot.js]: can you fix the issues instead of disabling it?

**Comment by geekgonecrazy** [comment]
@sampaiodiego agreed! If the scripts aren't there and it tries to load them what happens?  If hubot breaks badly... then we may want to add a migration that either turns off the internal hubot if scripts that used to be there are referenced... or remove them.

**Comment by ggazzo** [comment]
@geekgonecrazy the `~load~` are in a try catch so if the file/path doesnt exists , the code just put a message on console.

**Comment by geekgonecrazy** [comment]
@ggazzo :+1: ok awesome probably safe to leave then.  But we probably should add [Break] just so those that look at release notes before upgrading will see

**Pull request merged**